Directory.Build.props

```diff
@@ -3,11 +3,14 @@
   <PropertyGroup Label="Target Platforms" >
     <NetCoreVersions>netcoreapp3.1</NetCoreVersions>
     <NetStandardVersions>netstandard2.0</NetStandardVersions>
-    <NetFrameworkVersions>net472</NetFrameworkVersions> <!-- Framework version that should be targeted alongside with .net standard -->
-    <LibraryTargetFrameworks>$(NetCoreVersions);$(NetStandardVersions);$(NetFrameworkVersions)</LibraryTargetFrameworks>
-    <LibraryTargetFrameworks Condition="'$(OS)' != 'Windows_NT'">$(NetCoreVersions);$(NetStandardVersions)</LibraryTargetFrameworks>
+    <LibraryTargetFrameworks>$(NetCoreVersions);$(NetStandardVersions)</LibraryTargetFrameworks>
     <ExecutableTargetFrameworks>$(NetCoreVersions)</ExecutableTargetFrameworks>
-</PropertyGroup>
+  </PropertyGroup>
+  <PropertyGroup Label="UnitTest Targets">
+    <NetFameworkVersion>net472</NetFameworkVersion>
+    <UnitTestTargetValues>$(NetCoreVersions);$(NetFameworkVersion)</UnitTestTargetValues>
```

Comment:
Given that the non-unit test version of this is `LibraryTargetFrameworks`, I think `UnitTestTargetFrameworks` might be a better name here

Reply:
Makes sense, thank you